gradle/projects.main.properties

```diff
@@ -1,5 +1,6 @@
 nessie-bom=bom
 nessie-client=clients/client
+nessie-compatibility-engine=compatibility/test-engine
```

Comment:
Wouldn't this fit better in `testing/` - and called `nessie-multi-env-engine`, incl. package renames?

Reply:
sgmt - will move

Reply:
moved